api/controllers/v0.3/endpoints.js

```diff
@@ -17,7 +17,7 @@ const {
   errorResponseDataWithTag,
 } = require("../../helpers/responses");
 const DependentAccountAPI = require("./DependentAccountAPI");
-const { strToBool } = require("../../helpers/utils");
```

Comment:
What was this doing; why don't we need it anymore?

Reply:
If the client passes booleans as strings, `"false"` or `"true"`, this converts it to the boolean value, but it wasn't used in this file. Just accidentally imported.